Makefile

```diff
@@ -0,0 +1,18 @@
+PROJECT = trails
+
+SHELL_DEPS = sync
+
+dep_sync =  git git://github.com/inaka/sync.git  0.1.3
+dep_xref_runner = git git://github.com/inaka/xref_runner.git 0.2.2
```

Comment:
This dep is not in any dep list anymore. It should be in `TEST_DEPS` and you should add the [xref_SUITE](https://gist.github.com/elbrujohalcon/4761744ffad45d404f5d)